wagtail/admin/rich_text/editors/draftail/__init__.py

```diff
@@ -63,9 +64,19 @@ def format_value(self, value):
 
         return self.converter.from_database_format(value)
 
+    def get_options(self):
+        chooserUrls = {
+            "pageChooser": reverse("wagtailadmin_choose_page"),
+            "externalLinkChooser": reverse("wagtailadmin_choose_page_external_link"),
+            "emailLinkChooser": reverse("wagtailadmin_choose_page_email_link"),
+            "phoneLinkChooser": reverse("wagtailadmin_choose_page_phone_link"),
+            "anchorLinkChooser": reverse("wagtailadmin_choose_page_anchor_link"),
+        }
+        return {**self.options, "chooserUrls": chooserUrls}
+
     def get_context(self, name, value, attrs):
         context = super().get_context(name, value, attrs)
-        context["widget"]["options_json"] = json.dumps(self.options)
+        context["widget"]["options_json"] = json.dumps(self.get_options())
```

Comment:
If we prepare a custom JSON encoder, we can use the JSON approach here.

```suggestion
        context["widget"]["options_json"] = json.dumps(self.get_options(), cls=LazyStringEncoder)
```

Reply:
just 👍  because this addition `cls=LazyStringEncoder`  solved the entire problem.